src/bin/roundcorner.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(dead_code)]
+
```

Comment:
For the final "release" code, there shouldn't be any dead code, so not a good idea to allow it in general.

Reply:
I agree, I used it because there were many unused functions in that file and that seemed cleaner than prefixing them all with an underscore, but I can do that instead.

Reply:
done